src/detect-base64-decode.c

```diff
@@ -233,6 +233,9 @@ static int DetectBase64DecodeSetup(DetectEngineCtx *de_ctx, Signature *s,
         data->bytes = BASE64_DECODE_MAX;
     }
     if (data->bytes > de_ctx->base64_decode_max_len) {
+#ifdef FUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION
```

Comment:
should we enforce a limit for the non-fuzz case as well?

Reply:
Well, this comes from the signature...
Maybe there is a use case to get it big.
Or we can limit it to `BASE64_DECODE_MAX`
As you feel...